modules/calib3d/src/usac/bundle.cpp

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) 2020, Viktor Larsson
```

Comment:
What is origin of the file? Does does it make sense to place it as library to 3rd-party? The license text is not Apache 2.0 and we need to mentioned the library, author and license in OpenCv distributions.

Reply:
Discussed the issue on OpenCV core meeting. Decided to stay it as is. Sory for disturbance.